src/test/java/io/federecio/dropwizard/swagger/selenium/auth/DefaultServerWithAuthenticationSeleniumTest.java

```diff
@@ -17,6 +17,7 @@
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 import org.junit.ClassRule;
+import org.junit.Ignore;
```

Comment:
I re-enabled it. I wanted to check if it was the only one. Then I rebased with the upstream version because you had some more commits and it worked.